### DISCUSSION TITLE
How to add new slider to available commands?

### DISCUSSION THREAD

**Discussion by earthsworld** [rsjaffe/MIDI2LR#737] [General]
I'd love to add the new Curve refine sat slider to the commands list. 
Is that a difficult task if i know nothing about code and compiling?

**Comment by remov-b4-flight** [comment]
Edit Client.lua & Database.lua 
Delete MenuTrans.xml
Relaunch MIDI2LR application

**Comment by rsjaffe** [comment]
It's in the pending release: see https://github.com/rsjaffe/MIDI2LR/discussions/759#discussion-5924718
